Bender.yml

```diff
@@ -85,28 +86,6 @@ export_include_dirs:
   - hw/snitch_ssr/include
 
 sources:
-  # future
-  - files:
-      # Level 0
-      - hw/future/src/mem_to_axi_lite.sv
-      - hw/future/src/idma_reg64_frontend_reg_pkg.sv
-      - hw/future/src/idma_tf_id_gen.sv
-      - hw/future/src/dma/axi_dma_data_path.sv
-      - hw/future/src/axi_interleaved_xbar.sv
-      # Level 1
-      - hw/future/src/axi_zero_mem.sv
-      - hw/future/src/idma_reg64_frontend_reg_top.sv
-      # Level 2
-      - hw/future/src/idma_reg64_frontend.sv
-      - hw/future/src/dma/axi_dma_data_mover.sv
-      - hw/future/src/dma/axi_dma_burst_reshaper.sv
-      # Level 3
-      - hw/future/src/dma/axi_dma_backend.sv
-  - target: test
-    files:
-      - hw/future/test/fixture_axi_dma_backend.sv
-      - hw/future/test/tb_axi_dma_backend.sv
-
```

Comment:
Can we move the `interleaved_xbar` and `zero_mem` removal to another PR, such that it's easier to align with Occamy?